### PR TITLE
EXP-238 feat(linkme): add tab "teams" to settings

### PR DESCRIPTION
**Pull request by allangrds**
##  Contexto
Removendo trava que não mostrava a aba "Equipe" para empresas Link.me

## Como testar?
- [ ] Acessar dash com company Link.me, item "configurações" do menu e ver a aba "equipe";
- [ ] Na listagem de pessoas da equipe, na coluna "Permissões" só deve aparecer: Administrador, Somente leitura e vendedor;
- [ ] Na criação de um usuário, na coluna "Permissões" só deve aparecer: Administrador, Somente leitura e vendedor.

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b931fb3d180adde14f67f0ab5067030aac7116f1
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b931fb3d180adde14f67f0ab5067030aac7116f1
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b931fb3d180adde14f67f0ab5067030aac7116f1
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b931fb3d180adde14f67f0ab5067030aac7116f1
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: packages/pilot/src/containers/Settings/Company/TeamInfoTab/TeamManagement/UserTable.js [hunk 1]
```diff
@@ -170,6 +179,7 @@ class UserTable extends React.Component {
             />
           )
         }
+        asdasdsadsadsa
```
Comment (marisamodolo) [inline packages/pilot/src/containers/Settings/Company/TeamInfoTab/TeamManagement/UserTable.js:182]: 👀

File: packages/pilot/src/containers/Settings/Company/TeamInfoTab/TeamManagement/AddNewUserModal.js [hunk 2]
```diff
@@ -65,6 +52,30 @@ class AddNewUserModal extends React.Component {
     }
   }
 
+  getPermissions () {
```
Comment (vagnervst) [inline packages/pilot/src/containers/Settings/Company/TeamInfoTab/TeamManagement/AddNewUserModal.js:55]: Essa função poderia receber `t` para não precisar fixar os textos e puxa-los das translations

File: packages/pilot/src/containers/Settings/Company/TeamInfoTab/TeamManagement/UserTable.js (hunk 1 above)
Comment (vagnervst) [reply packages/pilot/src/containers/Settings/Company/TeamInfoTab/TeamManagement/UserTable.js:182]: aqui ainda não foi resolved @allangrds

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 182e74841ec989efd50ca9f88332be49545a3764
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 182e74841ec989efd50ca9f88332be49545a3764
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 182e74841ec989efd50ca9f88332be49545a3764
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 182e74841ec989efd50ca9f88332be49545a3764
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: packages/pilot/src/containers/Settings/Company/TeamInfoTab/TeamManagement/AddNewUserModal.js [hunk 3]
```diff
@@ -65,6 +66,19 @@ class AddNewUserModal extends React.Component {
...
+    const { company, t } = this.props
+    const permissionsList = getPermissionList(company)
+    const permissionsValues = permissionsList.map(item => (
+      {
+        name: t(`models.user.permission.${item}`),
+        value: item,
```
Comment (marisamodolo) [inline packages/pilot/src/containers/Settings/Company/TeamInfoTab/TeamManagement/AddNewUserModal.js:75]: Aqui ao tentar convidar um novo usuário tá dando erro, pq tá vindo com valor `seller`
image

File: packages/pilot/src/containers/Settings/Company/TeamInfoTab/TeamManagement/AddNewUserModal.js [hunk 4]
```diff
@@ -16,27 +16,28 @@ import IconInfo from 'emblematic-icons/svg/InfoCircle32.svg'
...
-    value: 'read_write',
-  },
-]
+const getPermissionList = (company) => {
+  const permissionsList = [
+    'admin',
+    'read_only',
+  ]
+  const newArrayValue = isPaymentLink(company)
+    ? 'seller'
```
Comment (vagnervst) [inline packages/pilot/src/containers/Settings/Company/TeamInfoTab/TeamManagement/AddNewUserModal.js:33]: isso faz com que mandemos `seller` para a API, o que não é uma permissão válida:

image

pensei em fazer uma função `getName` para ser usada no `getPermissions`, assim só traduzimos a label e mantemos o valor esperado pelo core:

```js
const getName = uncurryN(2, t => cond([
  [both(propEq('permission', 'read_write'), isPaymentLink), always(t('models.user.permission.seller'))],
  [T, ({ permission }) => t(`models.user.permission.${permission}`)]
]))
```

Essa função recebe como parâmetro:

```js
getName(t, {
  permission: 'read_only',
  company: {
    type: 'payment_link_app'
  }
})
```

[Link da implementação dela no try ramda](https://ramdajs.com/repl/#?const%20permissions%20%3D%20%5B%0A%20%20%27read_only%27%2C%0A%20%20%27read_write%27%2C%0A%20%20%27admin%27%2C%0A%5D%0A%0Aconst%20isPaymentLink%20%3D%20pathEq%28%5B%27company%27%2C%20%27type%27%5D%2C%20%27payment_link_app%27%29%0A%0Aconst%20getName%20%3D%20uncurryN%282%2C%20t%20%3D%3E%20cond%28%5B%0A%20%20%5Bboth%28propEq%28%27permission%27%2C%20%27read_write%27%29%2C%20isPaymentLink%29%2C%20always%28t%28%27models.user.permission.seller%27%29%29%5D%2C%0A%20%20%5BT%2C%20%28%7B%20permission%20%7D%29%20%3D%3E%20t%28%60models.user.permission.%24%7Bpermission%7D%60%29%5D%0A%5D%29%29%0A%0AgetName%28identity%2C%20%7B%0A%20%20permission%3A%20%27read_only%27%2C%0A%20%20company%3A%20%7B%0A%20%20%20%20type%3A%20%27payment_link_app%27%0A%20%20%7D%0A%7D%29)

**Pull request merged**